Stock_Alert/main.py

```diff
@@ -1,74 +1,66 @@
-import requests
+from response import Response
+from decimal import Decimal
 import os
 
 STOCK_SYMBOL = "OTLY"
 COMPANY_NAME = "Oatly"
 
 STOCK_ENDPOINT = "https://www.alphavantage.co/query"
-NEWS_ENDPOINT = "https://newsapi.org/v2/everything"
-
 stock_api_key = os.environ.get("STOCK_API_KEY")
+
+NEWS_ENDPOINT = "https://newsapi.org/v2/everything"
 news_api_key = os.environ.get("NEWS_API_KEY")
 
 # Get hold of the API endpoint of Alphavantage.
-stock_params = {
```

Comment:
I would prefer to leave this minor style as it is, because otherwise I can't outsource the Error Handling to a class as you suggested in your first feedback.  To handle the potential errors while making a GET request, I need to use the request.get() method already in the class and pass in the endpoint with the params AFAIK.

Reply:
@sasha-cz I suggest you have a variable `stock_params` defined as it was previously, and then you pass the variable to the method call instead of listing all params in the parentheses, it'd be more readable

Reply:
ok